test-dynamic-import.ts

```
@@ -0,0 +1,5 @@
+// This is a test file to reproduce the issue with no-restricted-imports
+import { useMessage } from './components/messaging/message/MessageContext.dynamic';
```

ESLint

[error] 'useMessage' is defined but never used.

(@typescript-eslint/no-unused-vars)